mac/deploy_mac.sh

```diff
@@ -47,15 +61,30 @@ build_app()
     local job_count
     job_count=$(sysctl -n hw.ncpu)
 
+    # Get Jamulus version
+    local app_version="$(cat "${project_path}" | sed -nE 's/^VERSION *= *(.*)$/\1/p')"
+
     make -f "${build_path}/Makefile" -C "${build_path}" -j "${job_count}"
 
     # Add Qt deployment dependencies
-    if [[ -z "$cert_name" ]]; then
+    if [[ -z "$macapp_cert_name" ]]; then
         macdeployqt "${build_path}/${target_name}.app" -verbose=2 -always-overwrite
     else
-        macdeployqt "${build_path}/${target_name}.app" -verbose=2 -always-overwrite -hardened-runtime -timestamp -appstore-compliant -sign-for-notarization="${cert_name}"
+        macdeployqt "${build_path}/${target_name}.app" -verbose=2 -always-overwrite -hardened-runtime -timestamp -appstore-compliant -sign-for-notarization="${macapp_cert_name}"
+        # verify signature
+        codesign -dv --verbose=4 "${build_path}/${target_name}.app"
+    fi
+
+    ## Build installer pkg file - for submission to App Store
+    if [[ -z "$cert_name" ]]; then
+        productbuild --component "${build_path}/${target_name}.app" /Applications "${build_path}/Jamulus_${app_version}.pkg"
+    else
+        productbuild --sign "${cert_name}" --keychain build.keychain --component "${build_path}/${target_name}.app" /Applications "${build_path}/Jamulus_${app_version}.pkg"        
     fi
-    mv "${build_path}/${target_name}.app" "${deploy_path}"
+
+    # move things
```

Comment:
Which things?You mean dmg file or pkg file?

Reply:
I'll clarify the comments here